clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/view/AzureVMImageLookupController.groovy

```diff
@@ -61,15 +62,31 @@ class AzureVMImageLookupController {
   List<AzureNamedImage> getVMImage(@PathVariable String account, @PathVariable String region, @PathVariable String imageId) {
```

Comment:
Our first approach to this is to start with an expected hard coded image name at first until we can figure out how to get Jenkins to pass the real thing. Implementation of how the image will be built and get dropped into the Azure Storage will follow after. The trigger is supposed to be a very simple one for now, just to notify the app when a new version of the image got uploaded in order to trigger the pipeline; I'm still working on the details so I can't be more specific at the moment.